src/report.tex

```diff
@@ -2143,10 +2160,30 @@ \subsection{Conclusion}
 activation in the region. It is suggested that the front 50 cm of the B$_4$C
 drawers could be removed, reducing the weight of the port plug without
 significantly impacting the shielding capability.
-
-%\newpage
-%\clearpage
-%\section{Acceptance Criteria}
+\subsection{Estimation of Uncertainties}
```

Comment:
none of these mention the statistical uncertainty of the neutron transport which we also can't propagate